pkg/controller/core.oam.dev/v1alpha2/application/suite_test.go

```diff
@@ -122,11 +124,12 @@ var _ = BeforeSuite(func(done Done) {
 	pd, err := definition.NewPackageDiscover(cfg)
 	Expect(err).To(BeNil())
 	reconciler = &Reconciler{
-		Client: k8sClient,
-		Log:    ctrl.Log.WithName("Application-Test"),
-		Scheme: testScheme,
-		dm:     dm,
-		pd:     pd,
+		Client:   k8sClient,
+		Log:      ctrl.Log.WithName("Application-Test"),
+		Scheme:   testScheme,
+		dm:       dm,
+		pd:       pd,
+		Recorder: record.NewFakeRecorder(10),
```

Comment:
what's the `10` mean? Will it affect our CI?

Reply:
this is event channel size, maybe it affects the ci, I will set a larger num.